interfaces/policy/helpers.go

```diff
@@ -221,6 +221,36 @@ func checkSlotConnectionConstraints(connc *ConnectCandidate, cstrs []*asserts.Sl
 	return firstErr
 }
 
+func checkSnapTypeSlotInstallationConstraints1(ic *InstallCandidateMinimalCheck, slot *snap.SlotInfo, cstrs *asserts.SlotInstallationConstraints) error {
```

Comment:
Why the trailing `1` in the function name?

Reply:
Fair question but I don't know the answer :). @pedronis may know, I guess it's simply setting stage for more of these checks in the future. FWTW this is consistent style across these functions.

Reply:
it's probably not a super common naming convention, I don't remember where I picked it up, but basically  if one has a function `foo` that operates on an array or list of elements,  the helper function, if it makes sense, that operates on a single element gets called `foo1`